internal/rbd/controllerserver.go

```diff
@@ -384,38 +409,53 @@ func (cs *ControllerServer) createBackingImage(ctx context.Context, cr *util.Cre
 	return nil
 }
 
-func (cs *ControllerServer) checkSnapshotSource(ctx context.Context, req *csi.CreateVolumeRequest,
-	cr *util.Credentials) (*rbdSnapshot, error) {
+func (cs *ControllerServer) checkContentSource(ctx context.Context, req *csi.CreateVolumeRequest, cr *util.Credentials) (*rbdVolume, *rbdSnapshot, error) {
```

Comment:
This function does not use `cs *ControllerServer`, so it should most likely not be a method of that type.
(It was already like this before, so no need to change, but should be a cleanup later)